### DISCUSSION TITLE
nixos WSL + root / tmpfs

### DISCUSSION THREAD

**Discussion by segator** [nix-community/NixOS-WSL#334] [Q&A]
I wonder if could be possible to have / tmpfs so as soon you reboot the WSL any change not done in the persistence volumes with https://nixos.wiki/wiki/Impermanence is lost.

I would like to define a corporate nixos WSL image for all the developers within my company using windows.

Thanks great project!

**Comment by K900** [comment]
Not easily, at least. The rootfs is actually mounted by the WSL container tooling, not by NixOS itself.
